resources/views/tasks/edit.blade.php

```diff
@@ -223,6 +223,7 @@ class="d-inline-flex pull-left align-items-center"
     const task = @json($task);
     const userHasAccessToTask = {{ Auth::user()->can('update', $task) ? "true": "false" }};
     const userIsAdmin = {{ Auth::user()->is_administrator ? "true": "false" }};
+    const userIsProcessManager = {{ Auth::user()->id === $task->process()->first()?->manager_id ? "true": "false" }};
```

Comment:
`$task->process()->first()` always executes the query to get the process row, maybe you could consider to use `$task->process`

Reply:
Considre to use `$task->request->processVersion` to get the right process version

Reply:
First comment fixed
POs recommend to to use the current Process Manager, so I fixed using directly ->process
https://processmaker.atlassian.net/browse/FOUR-6660?focusedCommentId=292519